TerriaJS/terriajs
# Add support for ncWMS2 `GetTimeseries`

### Add support for ncWMS2 `GetTimeseries`

Fixes https://github.com/TerriaJS/terriajs/issues/1424

This adds a new `supportsGetTimeseries` trait, which when true will replace `GetFeatureInfo` with `GetTimeseries` requests. It will also change `info_format` to `text/csv`, and show a chart in the feature info panel. Servers which advertise `GetTimeseries` capability will have this trait set to true by default. `GetTimeseries` requests will have `time = ""`.

To test

- http://ci.terria.io/get-timeseries/
- Test WMS (keep in mind that some layers have over a 1000 time slices, so responses may be slow)
  - ~https://psl.noaa.gov/thredds/wms/Datasets/ncep.reanalysis/Monthlies/tropopause/air.mon.mean.nc?service=WMS&version=1.3.0&request=GetCapabilities~ link is dead - see https://github.com/TerriaJS/terriajs/pull/6884#issuecomment-1748042961
- Load up a layer
- Click on the map
- Timeseries chart!

### Checklist
